.npmrc

```diff
@@ -1,5 +1,13 @@
-#save exact versions of dependencies from the package.json
+# https://docs.npmjs.com/cli/v8/using-npm/config
+
+# Save dependencies to package.json with an exact version rather than using npm's default semver range operator
 save-exact=true
 
-#enforces usage of the minimum node version in our .nvmrc
-engine-strict=true
\ No newline at end of file
+# Refuse to install (or even consider installing) any package that claims to not be compatible with the current Node.js version.
+engine-strict=true
+
+# If a package cannot be installed because of overly strict peerDependencies that collide, it provides a way to move forward resolving the situation.
+# TODO (debug):
+#   npm ERR! While resolving: salesforcedx-vscode-soql@54.8.0
+#   npm ERR! Found: @typescript-eslint/parser@3.10.1
```

Comment:
does it suggest a version that works or what?

Reply:
Add the dev deps and removed the legacy peer deps flag